cas-server-support-oauth/src/test/java/org/jasig/cas/support/oauth/web/OAuth20AuthorizeControllerTests.java

```diff
@@ -43,6 +45,8 @@
     private static final String ID = "id";
     private static final String FIRST_NAME_ATTRIBUTE = "firstName";
     private static final String FIRST_NAME = "jerome";
+    private static final String LAST_NAME_ATTRIBUTE = "lastName";
+    private static final String LAST_NAME = "LELEU";
```

Comment:
I fixed some checktyle issues, then merged with master. Now I have the following errors when running tests: `org.springframework.context.annotation.ConflictingBeanDefinitionException: Annotation-specified bean name 'proxy20TicketUniqueIdGenerator' for bean class [org.jasig.cas.util.HostNameBasedUniqueTicketIdGenerator$ProxyTicketIdGenerator] conflicts with existing, non-compatible bean definition of same name and class [org.jasig.cas.util.DefaultUniqueTicketIdGenerator$ProxyTicketIdGenerator]`